packaging/src/docker/Dockerfile

```diff
@@ -14,14 +14,31 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
+ARG BUILD_ENV
+
 FROM ubuntu as unarchive
+ONBUILD COPY hadoop-*.tar.gz /opt
+ONBUILD COPY apache-hive-*-bin.tar.gz /opt
+ONBUILD COPY apache-tez-*-bin.tar.gz /opt
+
+FROM ubuntu as archive
+ARG HADOOP_VERSION
+ARG HIVE_VERSION
+ARG TEZ_VERSION
+ONBUILD RUN apt-get update && apt-get -y install wget
+ONBUILD RUN wget https://archive.apache.org/dist/tez/$TEZ_VERSION/apache-tez-$TEZ_VERSION-bin.tar.gz && \
+ wget https://archive.apache.org/dist/hadoop/core/hadoop-$HADOOP_VERSION/hadoop-$HADOOP_VERSION.tar.gz && \
+ wget https://archive.apache.org/dist/hive/hive-$HIVE_VERSION/apache-hive-$HIVE_VERSION-bin.tar.gz
```

Comment:
nit: When we create a rel/release-*, in Dockerfile we get the hive tarball from https://archive.apache.org/dist/hive/hive-$HIVE_VERSION/apache-hive-$HIVE_VERSION-bin.tar.gz, I'm not sure it would be there on the event rel/release-* created.